### PR TITLE
Feature/favicon

### PR DESCRIPTION
**Pull request by scovl**
Adding `favicon.ico` to the `example` directory and handling it in the code:  https://github.com/rochacbruno/marmite/issues/11

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -130,6 +130,34 @@ fn main() -> io::Result<()> {
...
     }
 
+    // Copy or create favicon.ico
+    let favicon_dst = output_folder.join("favicon.ico");
+
+    // Possible paths where favicon.ico might exist
+    let favicon_src_paths = [
+        input_folder.join("static").join("favicon.ico"),           // User's favicon.ico
+        Path::new("example/static/favicon.ico").to_path_buf(),     // Default favicon.ico in project
+    ];
```
Comment (rochacbruno) [inline src/main.rs:140]: This path assumes `marmite` will always execute from the repository rootm but
when user installs from `cargo install marmite` or downloads the binary directly, they will have only the `marmite` binary, and optionally the `input_folder/static/favicon.ico`.

On #20 I want to embed all the essential files from `example` in the binary, for now we may have only one possible favicon location: 

```suggestion
    let favicon_src_paths = [
        input_folder.join("static").join("favicon.ico"),           // User's favicon.ico
        // on #20 we may have embedded statics
    ];
```

File: src/main.rs [hunk 2]
```diff
@@ -96,7 +96,7 @@ fn main() -> io::Result<()> {
 
     robots::handle_robots(&content_dir, &output_path);
 
-    // Initialize Tera templates
+    // Intialize Tera templates
```
Comment (rochacbruno) [inline src/main.rs:99]:
```suggestion
    // Initialize Tera templates
```

**Review by rochacbruno — CHANGES_REQUESTED** [review]

File: src/main.rs (hunk 1 above)
Comment (scovl) [reply src/main.rs:140]: I hadn't paind attention on that, you`re right. I'll think of something better to resolve both issues.

File: src/main.rs (hunk 2 above)
Comment (scovl) [reply src/main.rs]: Can you believe I read it over and over and still didn't catch the error? My gosh...

**Review by rochacbruno — APPROVED** [review]

**Pull request merged**